tests/scale/test_load.py

```diff
@@ -46,7 +46,8 @@ def test_scaling_load(master_count,
                       run_delay,
                       cpu_quota,
                       work_duration,
-                      mom):
+                      mom,
+                      external_volume):
```

Comment:
I think this may be passed through as a string (`"True"`). This could cause issues on some `if` checks.